.github/workflows/e2e.yaml

```diff
@@ -9,27 +9,85 @@ env:
   SHOREBIRD_TOKEN: ${{ secrets.SHOREBIRD_TOKEN }}
 
 jobs:
-  verify_cli_installation:
+  changes:
```

Comment:
I don't think we need this for the e2e because we probably won't end up having a complex matrix of jobs. We can probably keep it simple and only have it trigger on path changes at the workflow level.

Reply:
Fair point. If we're running every day anyways, I'll just hardcode `packages/shorebird_cli`